src/moscot/base/output.py

```diff
@@ -168,6 +171,93 @@ def chain(self, outputs: Iterable["BaseSolverOutput"], scale_by_marginals: bool
 
         return op
 
+    def sparsify(
+        self,
+        mode: Literal["threshold", "percentile", "min_row"],
+        threshold: Optional[float] = None,
+        batch_size: int = 1024,
+        n_samples: Optional[int] = None,
+        seed: Optional[int] = None,
+    ) -> None:
+        """
+        Sparsify the :attr:transport_matrix.
+
+        This function sets all entries of the transport matrix below `threshold` to 0 and
+        returns the result as a :class:`~scipy.sparse.csr_matrix`.
+
+        Parameters
+        ----------
+        mode
+            Which threshold to use for sparsification. Can be one of:
+
+                - 'threshold' - threshold below which entries are set to 0.0.
+                - 'percentile' - determine threshold by percentile below which entries are set to 0. Hence, between 0
+                  and 100.
+                - 'min_row' - choose the threshold such that each row has at least one non-zero entry.
+
+        threshold
+            Threshold or percentile depending on ``mode```. If ``mode = 'min_row'``, ``threshold``` can be neglected.
```

Comment:
Extra \` in the mode.

Reply:
Also in threshold.

Reply:
slightly adapted the docs, so this does not apply any more